promdens.py

```diff
@@ -598,8 +595,9 @@ def windowing(self):
     add = ''
     if item == 'nsamples' and config[item] == 0:
         add = '(All input data will be used)'
-    print(f"  - {item:20s}: {config[item]}   {add}")
-print()
+    elif item in ['npsamples', 'seed', 'preselect', 'neg_handling'] and config['method'] == 'pdaw':
+        add = '(Parameter not necessary for PDAW)'
```

Comment:
Should we simply not print these parameters then?

Reply:
Done